gpytorch/kernels/grid_kron_wrapper.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+#TODO: clean this up
+import torch
+import math
+from .. import settings
+from . import GridKernel
+
+from ..lazy import InterpolatedLazyTensor, KroneckerProductLazyLogDet, NonLazyTensor, lazify, delazify, ToeplitzLazyTensor
+from ..models.exact_prediction_strategies import InterpolatedPredictionStrategy
+from ..utils.interpolation import Interpolation
+
+class GridKronWrapper(GridKernel):
```

Comment:
Is this class actually used anywhere? `WrappedSpectralKernel` is commented out above in the `kernels/__init__.py`.

Reply:
No I removed the `WrappedSpectralKernel` from the PR for now as it's code for scalability and not a base implementation.